api/src/app.module.ts

```diff
@@ -11,9 +11,17 @@ import { RecipesModule } from './recipes/recipes.module';
 import { GrocyModule } from './grocy/grocy.module';
 import { ImportModule } from './import/import.module';
 
+const mongoUser = process.env.MONGO_USER;
+const mongoPassword = process.env.MONGO_PASSWORD;
+const mongoHost = process.env.MONGO_HOST;
+const mongoPort = process.env.MONGO_PORT;
+const mongoConnectionString = `mongodb://${mongoUser}:${mongoPassword}@${mongoHost}:${mongoPort}/`;
```

Comment:
```suggestion
const mongoConnectionString = process.env.MONGO_URI || `mongodb://${mongoUser}:${mongoPassword}@${mongoHost}:${mongoPort}/`;
```
Passing a MONGO_URI is a common industry practice, additionally this code could use some validation/throwing (f.e. in case it gets run outside of docker compose)